### ISSUE TITLE
Selecting the draggable handles of a block then typing does nothing

### ISSUE THREAD

**Issue by tomjn** [WordPress/gutenberg#8541]
**Describe the bug**
I clicked insert block, and selected the block with the cursor, then attempted to type `hello` but nothing happened.

**To Reproduce**
1. Click insert block
2. ~Select the newly created paragraph block with the mouse~ Click on the draggable handle
3. try to type
4. nothing is shown on screen

**Expected behavior**
I expected the inserter UI to dissapear and the word `hello` to show

**Screenshots**
<img width="685" alt="screen shot 2018-08-05 at 01 50 15" src="https://user-images.githubusercontent.com/58855/43681656-3af88ebe-9852-11e8-9413-e5b811bd006e.png">

**Desktop (please complete the following information):**
 - OS: MacOS High Sierra
 - Browser Chrome
 - Version 68

**Additional context**
- Replicated on Testgutenberg.com

**Comment by tomjn** [comment]
I saw testgutenberg.com runs v3.3, so I tested on frontenberg which is 3.4 with the same results

**Comment by abwdvm** [comment]
I was not able to reproduce this error on Testgutenberg.com  When I add block, paragraph, the flashing i beam is ready to enter text.
Chrome Version 68.0.3440.84 (Official Build) (64-bit)
Windows 7, Home Prem, SP 1

**Comment by tomjn** [comment]
@abwdvm did you select the block though? Clicking insert then immediatley typing works, but I selected the block with my mouse cursor as shown in the screenshot

**Comment by designsimply** [comment]
Tested and confirmed that I cannot interact with a block via keyboard while in drag-and-drop mode using WordPress 4.9.8 and Gutenberg 3.4.0 with Chrome 67.0.3396.99 on macOS 10.13.6 ([59s](https://cloudup.com/cHC2hSHAFhq)).

![draggable-handles-example](https://user-images.githubusercontent.com/1119271/43746432-7cc0e012-99a1-11e8-8cc6-67919e231827.png)
<sup>Seen at http://alittletestblog.com/wp-admin/post.php?post=14134&action=edit running WordPress 4.9.8 and Gutenberg 3.4.0 using Chrome 67.0.3396.99 on macOS 10.13.6.</sup>

I updated step 2 from "Select the newly created paragraph block with the mouse" to "Click on the draggable handle" to help clarify.

Possibly related: #7114.

**Comment by nosolosw** [comment]
I think the fact that the draggable handle was invisible led to this kind of odd behavior/bug, as the user may have not been aware that the click happened in the draggable handle. Now that the draggable handle has its own icon within the block movers the expectations are clear: if you click on it, as with any other button, you'll lose focus on the block. With that in mind, I'm closing this.

**Comment by tomjn** [comment]
@nosolosw my original issue was that if you selected a paragraph block then typed, nothing was typed. This issue with the dragging is just a subset of that problem. It's possible to select a block without using the draggable handle

**Comment by tomjn** [comment]
@nosolosw For example, if I press tab to arrive at a block like this:

<img width="715" alt="screenshot 2018-10-16 at 13 44 59" src="https://user-images.githubusercontent.com/58855/47017279-bf790f80-d149-11e8-9b78-e06eb3034437.png">

Typing does nothing, but I expect instead to see a caret appear and for what I type to become visible. Instead I have to either explicitly click, or tab through the entire toolbar. That's a lot of UI friction

Additionally, I clicked back to this github issue tab, and when I clicked back, and pressed tab, it went to the right hand sidebar.

Additionally, it's possible to click a non-draggable part of the inserter block, leaving the user in a situation where the block is selected but nothing can be typed:

<img width="865" alt="screenshot 2018-10-16 at 13 48 01" src="https://user-images.githubusercontent.com/58855/47017424-20a0e300-d14a-11e8-8091-0f5bfc7ec1f5.png">

Notice the lack of caret, and the missing selection highlight. Pressing tab brings it into focus. To do this, place the cursor above the block and move it upwards or downwards until the text cursor becomes a normal arrow cursor, and click.
